### PR TITLE
Combat2.0 merge TO combat2.0 superbranch

### PR DESCRIPTION
**Pull request by 1sourcecontrol**
This is a merge (to the combat 2.0 superbranch)  of what I have so far of the basic rule changes for combat 2.0. 

Major Changes under:
>Critical Hits
Armor
Fighting
Hip Firing
Blind Firing

Please review for clarity, feasibility, and/or goodness of the new/changed features. This would be a good time to talk about any issues you have.

### CODE REVIEW

File: play/PlayerRules/Basic Rules.txt [hunk 1]
```diff
@@ -239,7 +239,7 @@ points for the level. Additionally, any stat point alteration forces a recalulat
 ╠═══════╬══════════╬═══════╬════════════╬════════════╬═══════════════╣
 ║ 14    ║ "        ║ 8     ║            ║            ║ 11            ║
 ╠═══════╬══════════╬═══════╬════════════╬════════════╬═══════════════╣
-║ 15    ║ "        ║       ║ 3          ║ +1         ║               ║
+║ 15    ║ "        ║       ║ 3          ║ +1         ║               ║
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt:295]: Lol! Whitespace.

File: play/PlayerRules/Basic Rules.txt [hunk 2]
```diff
@@ -371,23 +371,23 @@ Characters also incur "penalty attacks" when moving through or next to an
...
-damage dealt is based on your DEX or STR modifier. If you are weilding a 
-weapon against an unarmed opponent, they may still incur damage depending on 
-their skill.
+damage dealt is based on a combination of your DEX and STR modifier. If you 
+are weilding a weapon against an unarmed opponent, they may still incur damage
+depending on their skill.
 A character may grapple another character when they are both in melee 
 distance. Doing so provides the grappled player a "penalty attack" on the 
-graPpling player. During grappling, both sides roll opposing grapple checks to 
+grappling player. During grappling, both sides roll opposing grapple checks to
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: Thanks for spell checking. Hopefully by doing reviews like this one we'll end up leaving less spelling errors to clean up. Spelling errors also aren't just yours to clean. Trowl and I will fix the ones that we see also.

File: play/PlayerRules/Basic Rules.txt [hunk 3]
```diff
@@ -371,23 +371,23 @@ Characters also incur "penalty attacks" when moving through or next to an
...
+to subdue your opponent (by choke-out, pin, or a joint lock). You may also strike
+an opponent during grappling, but if the strike fails your opponent may attempt 
+to gain control of the grapple. Skill in grappling is different than skill in 
+unarmed combat.
 
 Actions:
-During combat, your character is given 4 Actions to use for fighting, 
-movement, using items, etc. Here is a table of common tasks and their
+During combat, your character has 4 Actions. Actions are used for fighting, 
+movement, items, whimpering, etc. Here is a table of common tasks and their
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt:448]: whimpering? Why was this added?

File: play/PlayerRules/Basic Rules.txt [hunk 4]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
 
 Fighting:
 During combat, most weapons are Semi-Auto, meaning that each time you pull the 
-trigger, the gun fires and chambers a new round automatically one time. It 
+trigger, the gun fires once and chambers a new round automatically. It
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt:479]: Good clarification! :)

File: play/PlayerRules/Basic Rules.txt [hunk 5]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
 is the rechambering of the weapon.
-To hit, Players must roll a percentile that is over their weapon's % miss 
-stat. When an enemy is in cover, their cover score is added to your weapon's 
-miss chance. If your normal miss chance is 25, and the enemy is in full cover, 
-your miss chance is increased by 50, for a final DC of 75.
-Critical Hits:
-If your Luck is 5, on a roll of 90-1 through 90-5, the hit is considered a 
-Critical hit, and deals an extra 10 damage. on a roll of 90-6 through 00-0, 
-the hit is considered an Extra-Critical hit, and deals an extra 20 damage. 
+To hit a target, Players must roll a d10 that is over their weapon's miss stat.
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: This is a fairly radical change. I know you did it to simplify play and speed up turns. I'm approving the change but I'm still uneasy about it.

File: play/PlayerRules/Basic Rules.txt [hunk 6]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+shoot more than 1 bullet per Action by default).
 
 Gun Jams:
-On a roll of 00-01, the weapon Jams, necessitating a weapon skill check, DC 60.
-When a player is defending against a hit, they roll Percentiles under their 
-armor in order to apply a Damage Reduction, or to apply damage to their 
-Capacitive Sheilds if the player has them.
+If at any time that you are firing a gun and you roll a weapon accuracy of 1, roll a second d10.
+If that second d10 roll is a 1 or 2, the weapon Jams, necessitating a weapon 
+skill check, DC 60.
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: Flat weapon unjam? Instead of unjam per weapon? ... Actually now I think about it, every time this has happened, it has slowed the game down substantially. I think I like it.

File: play/PlayerRules/Basic Rules.txt [hunk 7]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+skill check, DC 60. 
 
-Hip Firing doubles a weapon''s miss chance. It is much smarter to takes one 
-action to Aim, and spend the rest of your shots on firing. Its also a great 
-idea to take an action at the end of your turn to "hunker," and gain a 25% 
+Armor and Getting Hit:
+When a player is defending against a hit, they roll Percentiles under their 
+armor in order to see if their armor blocked the bullet, and from there, how much
+damage they receive. If the player has no armor, they roll for a target 50 Percentile. 
+If the shot/attack rolls above 50%, they receive half-damage. Below 50%, full damage.
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: Is this the roll the attacker makes to hit? Or the roll that someone who is hit makes to see where/if their armor is hit? The wording "shot/attack rolls" seems like "attack roll". Could we change it to "If you get above a 50% on defense roll" or something similar?

File: play/PlayerRules/Basic Rules.txt [hunk 8]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+
+Armor consists of 3 concepts, Armor Points, Armor Piercing (AP) level, and % Coverage.
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: Ambiguity between initials of Armor Piercing and Armor Points. Please rename.

File: play/PlayerRules/Basic Rules.txt [hunk 9]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+Starting from the end, Coverage is simply the percentile that you have to roll
+under to see if armor was hit. If the attack misses armor, or rolls over 50%, it does
+half damage. If the attack hits armor: AP level comes into play.
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: Please state explicitly in the armor rules that missing armor does half damage because the bullet is hitting a less vital part of the body. Otherwise this effect is counterintuitive.

File: play/PlayerRules/Basic Rules.txt [hunk 10]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+
+Hip Firing:
+Add 3 to a weapon''s miss chance. You cannot use attachments to assist in
+hip firing unless otherwise noted. It is much smarter to takes one 
+action to Aim, and spend the rest of your shots firing. It's also a great 
+idea to take an action at the end of your turn to "hunker," and gain a +2 
 cover advantage.
+If your weapon has a minimum distance (usually because of a high-powered
+scope), and you fire inside that distance, you are forced to hip fire.
+
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: This explanation of hip firing is _much_ better in every way than our old explanation. I shipit hard.

File: play/PlayerRules/Basic Rules.txt [hunk 11]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+Blind Firing:
+Add 6 to a weapon's miss chance. Blind firing allows you to shoot around cover
+without looking or putting yourself at risk. Your blind fire accuracy is affected 
+by your base weapon miss chance(s), not affected by regular attachments, and only 
+reduced by LUCK and PER mod (max of -4 miss chance from LUCK mod + PER combat mod).
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: I also like that blindfiring is available to everybody. Please adapt the blindfire feat acordingly or assign that work to another dev and have them merge that to the superbranch.

File: play/PlayerRules/Basic Rules.txt [hunk 12]
```diff
@@ -418,49 +418,94 @@ You can jump vertically Str/3 meters.
...
+crouched, you have -1 movement speed per action (minimum 1) and you cannot
+sprint or use other movement abilities.
+	Going prone costs 1 Action, and standing up costs 1 Action. Lying flat down on
+your chest makes you a smaller target and helps you aim better, but also makes you
+easier to hit in melee. -0.5 gun miss chance while prone and +2 chance to be hit in
+melee. Being prone severely limits your movement: 1/2 movement speed while prone, 
+round-up; maximum 4 movement per 2 actions.
+	Hunkering costs 1 action, and you cannot fire your weapons. Imagine 
 curling up in a ball, to pose the absolute smallest target possible. Its a 
 free action to de-hunker
```
Comment (bleehu) [inline play/PlayerRules/Basic Rules.txt]: I absolutely love the new stance mechanics. Shipit. Shipit good.

**Review by bleehu — CHANGES_REQUESTED** [review]
Some clarifications around armor rules requested, and some other small issues that we could patch later. This is looking very good! Let's see if we can implement combat 2.0 quickly and a little dirty and polish it with our small patches to dev workflow that has been working well so far.

File: play/PlayerRules/Basic Rules.txt (hunk 2 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: Sounds good!

File: play/PlayerRules/Basic Rules.txt (hunk 3 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt:448]: I thought it was funny lol

File: play/PlayerRules/Basic Rules.txt (hunk 4 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt:479]: I thought it sounded better

File: play/PlayerRules/Basic Rules.txt (hunk 5 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: We'll try and test it asap

File: play/PlayerRules/Basic Rules.txt (hunk 6 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: I'm not sure what you mean... The intent of this was to exactly replicate the way jams are done in the percentile version. By, effectively, adding back in the one's place die roll.

File: play/PlayerRules/Basic Rules.txt (hunk 7 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: This is supposed to be just the armor roll. I'll do my best to reword it

File: play/PlayerRules/Basic Rules.txt (hunk 9 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: Alright!

File: play/PlayerRules/Basic Rules.txt (hunk 8 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: Alright!

File: play/PlayerRules/Basic Rules.txt (hunk 11 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: I plan to remove/change the blindfire feat as part of the combat overhaul

Comment (1sourcecontrol) [inline play/PlayerRules/Basic Rules.txt]: I plan on changing it with the rest of the combat 2.0 overhaul

**Comment by 1sourcecontrol** [comment]
@bleehu please revisit your comments, I replied to some

**Review by bleehu — APPROVED** [review]

File: play/PlayerRules/Basic Rules.txt (hunk 6 above)
Comment (1sourcecontrol) [reply play/PlayerRules/Basic Rules.txt]: @bleehu , before I shipit, can you clarify what you're talking about here?

Comment (bleehu) [reply play/PlayerRules/Basic Rules.txt]: In the old system some weapons (like pistols) were easier to reload than others (like light machineguns). losing that I think will simplify things. And we're losing complexity, but we never really got a lot of depth out of it. Nobody ever said "I'm picking this gun over that one 'cause it's easier to reload!"

**Comment by trowl223** [comment]
Looks good!
